nodes/mtdevice.py

```diff
@@ -1699,6 +1815,64 @@ def get_settings(arg):
     settings = timestamp | orient_mode | calib_mode | NED
     return settings
 
+def get_synchronization_settings(arg):
+    """Parse command line synchronization-settings argument."""
+    # Parse each field from the argument
+    settings = arg.split(',')
+    try:
+        # convert string to int
+        settings = tuple([int(i) for i in settings])
+    except ValueError:
+        print "Synchronization settings must be integers."
+        return
+    # check synchronization settings
+    if settings[0] in (3, 4, 8, 9, 11) and \
+       settings[1] in (0, 1, 2, 4, 5, 6) and \
+       settings[2] in (1, 2, 3) and \
```

Comment:
`settings[2]` can be also be `0`; that's actually the only way the clear the synchronization settings